src/main/java/io/vertx/core/net/impl/KeyStoreHelper.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2011-2021 Contributors to the Eclipse Foundation
+ * Copyright (c) 2011-2021 VMware, Inc. and others
```

Comment:
I believe this conforms to both my employer's guidelines and yours, but I'm a bit new to this - please let me know if anything is amiss here from your POV. Thanks.

Reply:
you need to sign the Eclipse Contributor Agreement to validate this PR and thus you become a contributor of the foundation.

Reply:
there is a check in the PR that says you are not covered by the ECA yet.

Reply:
I've signed the ECA and looks like the check is passing now.

Reply:
You need to keep `Contributors to the Eclipse Foundation`

Reply:
Technically this is what you are doing, you are a contributor to the Eclipse Foundation because you signed the ECA to contribute. This is per-foundation guidelines.

Otherwise we'd have every company adding their own name here, and it would be a mess.